src/main/java/org/jenkinsci/plugins/additionalmetrics/UnstableRateColumn.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * MIT License
+ *
+ * Copyright (c) 2022 Chadi El Masri, Oussama Ben Ghorbel
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+
+package org.jenkinsci.plugins.additionalmetrics;
+
+import com.google.common.collect.ImmutableList;
+import hudson.Extension;
+import hudson.model.Job;
+import hudson.model.Run;
+import hudson.views.ListViewColumn;
+import org.jenkinsci.Symbol;
+import static org.jenkinsci.plugins.additionalmetrics.Helpers.COMPLETED;
+import static org.jenkinsci.plugins.additionalmetrics.Helpers.NOT_SUCCESS;
+import static org.jenkinsci.plugins.additionalmetrics.Helpers.UNSTABLE;
+import static org.jenkinsci.plugins.additionalmetrics.Utils.rateOf;
+import org.kohsuke.stapler.DataBoundConstructor;
+
+public class UnstableRateColumn extends ListViewColumn {
+
+    @DataBoundConstructor
+    public UnstableRateColumn() {
+        super();
+    }
+
+    public Rate getUnstableRate(Job<? extends Job, ? extends Run> job) {
+        return rateOf(
+                job.getBuilds(),
+                ImmutableList.of(COMPLETED, NOT_SUCCESS),
+                UNSTABLE
+        ).orElse(null);
+    }
```

Comment:
So this is the rate of unstable / failed?
I had to read the code to figure it out :-)
When I read unstable rate, I think unstable / total.

What do you think?

Reply:
unstable / total it is.